exercises/concept/fruit-picker/.meta/exemplar.js

```diff
@@ -1,50 +1,46 @@
 /// <reference path="../global.d.ts" />
+//
 // @ts-check
+//
+// The lines above enable type checking for this file. Various IDEs interpret
+// the @ts-check and reference directives. Together, they give you helpful
+// autocompletion when implementing this exercise. You don't need to understand
+// them in order to use it.
+//
+// In your own projects, files, and code, you can play with @ts-check as well.
```

Comment:
I discussed with DJ that we only add the longer text in the first "layers" of concept exercises and then not any more afterwards so it can be removed.

Reply:
Yeah. By now we hope that people get this :)

Reply:
Sure

Reply:
Removed in https://github.com/exercism/javascript/pull/1725/commits/9538bc5dc0412bb4a00f5d5eb71e8a9eb7eaf297